package.json

```diff
@@ -92,8 +95,9 @@
 		"discord.js": "^12.5.3",
 		"dotenv-cra": "^2.0.2",
 		"gifencoder": "^2.0.1",
+		"google-protobuf": "^3.16.0-rc.1",
 		"he": "^1.2.0",
-		"ioredis": "^4.25.0",
+		"ioredis": "^4.26.0",
```

Comment:
Revert lest this is going to cause conflicts down the road, i.e. if not merged before sunday and depfu comes around.